Automattic/wp-calypso
# Openverse: Update icon to reflect the right size

#### Changes proposed in this Pull Request

* Updated Openverse icon to be the same size as the other external media icons.

#### Testing instructions



* Navigate to the media library on a test site (append `?flags=external-media/openverse` on Calypso Live)
* In the dropdown of external media see that Openverse icon is similar in size to the other icons.


#### Screenshots
Before | After
--- | ---
<img width="235" alt="Screen Shot 2022-03-24 at 4 02 16 PM" src="https://user-images.githubusercontent.com/1398304/160024394-37e4d425-596e-4cb2-be79-36812ca3f435.png"> | <img width="232" alt="Screen Shot 2022-03-24 at 4 04 00 PM" src="https://user-images.githubusercontent.com/1398304/160024398-67093eb7-9c62-4009-834b-7353c8cb6d50.png">

See #62221.
